src/main/java/org/apache/commons/collections4/ArrayUtils.java

```diff
@@ -38,6 +38,12 @@
  */
 class ArrayUtils {
 
+
+    /**
+     * Don't allow instances.
+     */
+    private ArrayUtils() {}
```

Comment:
This appears to be the only change that breaks backward compatibility.

Reply:
Ha! Thanks for your last comment @arturobernalg ! I did not see the class was private. I checked with clirr, it looks good.

```
kinow@ranma:~/Development/java/apache/commons-collections$ git log -n 1
commit 0679fa25d199ef6f276a430c43fcdecb1aa7c1c1 (HEAD -> pr-195)
Author: Arturo Bernal <arturobernalg@gmail.com>
Date:   Thu Nov 19 07:45:05 2020 +0100

    COLLECTIONS-772 - Provide private constructor to the Utils Collections
kinow@ranma:~/Development/java/apache/commons-collections$ mvn clirr:check
...
[INFO] --- clirr-maven-plugin:2.8:check (default-cli) @ commons-collections4 ---
[INFO] artifact org.apache.commons:commons-collections4: checking for updates from repo.jenkins-ci.org
[INFO] artifact org.apache.commons:commons-collections4: checking for updates from central
[INFO] Comparing to version: 4.4
[INFO] Succeeded with 0 errors; 0 warnings; and 49 other changes.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 12.449 s
[INFO] Finished at: 2020-11-21T22:15:37+13:00
[INFO] ------------------------------------------------------------------------

```

Reply:
Thank you @kinow

Reply:
Note that we need to switch to JApiCmp...